### ISSUE TITLE
Layout : leader lines + full bleed images

### ISSUE THREAD

**Issue by vns2** [janastu/ncbs-modular#138]
Layout for chapters needs leader lines and full bleed images as per design. 

![layout](https://user-images.githubusercontent.com/4581081/29066671-8cd4113a-7c4d-11e7-834a-412d4892dc53.png)

**Comment by Madhuvanthi-A** [comment]
@vns2  you should sit and do the leader lines, block quotes  and featured images.
we are low on resources.

**Comment by vns2** [comment]
Leader lines and 'featured' images is driven by design aesthetics, not by content at this stage. I thought this was what was worked on back in mid July?

**Comment by salus-sage** [comment]
Hypothesis group to curate the images and content from archives
https://hypothes.is/groups/EYei9VBb/ncbs-curation

**Comment by vns2** [comment]
I need some help with understanding this. What kind of information do I have to put on hypothesis for each page? This issue was primarily for full bleed or large images on each chapter. And the leader line design, etc. There are 24 chapters. I see 71 annotations on Hypothesis. For instance, attached image indicates that these two images are selected as full bleed for the same chapter? 

![hypothesis_annotations](https://user-images.githubusercontent.com/4581081/31173711-3ad52bca-a926-11e7-9c76-ba901f072489.png)

**Comment by salus-sage** [comment]
Yes, our team as chosen more than one option for each chapter. I guess it
will be easy to demo in person.
When you click on see annotations in context, then you will see the image
along with the anno. Then you can add your comment as a reply, and the
discussion can get on in this threads.

**Comment by vns2** [comment]
I started replies on hypothesis but I find this a little cumbersome. Here is the easier workflow for now, if that is agreeable to you. As default, use the images with tag ending in "-P1". http://archives.ncbs.res.in/items/tags. For instance, 1-Identity-P1, 1-Reflection-P1, and so on. 
That covers 23 out of 24 chapters. For remaining chapter: 4-Toggle, use the following as full image: http://archives.ncbs.res.in/items/browse?tags=4-Toggle-PS1-1

Now that leaves the question of placement of full image within the chapter. For this again, I suggest you use the same location as in the old website. Let me know if this works.

**Comment by vns2** [comment]
Can this issue be pushed ahead to milestone 1.0.8 instead?
